tests/compiler/function-call.optimized.wat

```diff
@@ -1257,6 +1257,11 @@
   local.get $0
   i32.const 20
   i32.add
+  local.tee $0
+  i32.const 0
+  i32.const 0
+  memory.fill
```

Comment:
This is an optimization artifact found in test cases where classes with zero fields are used. Inlines `size=0`, then preserving side effects I suppose, while previously the polyfill could be eliminated. Likely not worth worrying about.

Reply:
See this PR: https://github.com/WebAssembly/binaryen/pull/4295